Biz/Model/t/MailPartList/8.out

```diff
@@ -0,0 +1,19 @@
+use strict;
+use MIME::Base64;
+[
+    {
+        index => 0,
+	mime_type => 'x-message/rfc822-headers',
+	mime_entity => '',
+	'->get_header subject' => 'an image with an emdash in the filename',
+	'->get_header from' => 'Rob Nagler <nagler@bivio.biz>',
+	'->get_header date' => '2453757 65705',
+    },
+    {
+        index => 1,
+	mime_type => 'application/pdf',
+	'->get_file_name' => '2021 - Foo Bar.pdf',
+	'->format_uri_for_part SITE_ROOT' => '/2021%20-%20Foo%20Bar.pdf?p=1&t=1',
```

Comment:
Spaces should be converted to dashes and multiple dashes should be simplified. It's only necessary for the download rendering.